pages/cpm.vue

```diff
@@ -69,14 +87,41 @@ export default {
       if (!L.Browser.ie && !L.Browser.opera && !L.Browser.edge) {
         layer.bringToFront()
       }
+
+      this.info.update(layer._leaflet_id)
+      console.log(layer._leaflet_id)
     },
     resetHighlight (e) {
       const layer = e.target
       layer.setStyle({ fillColor: '#ffffff', weight: 2, opacity: 1, color: 'white', dashArray: '3', fillOpacity: 0.7 })
+      this.info.update()
     },
     zoomToFeature (e) {
       this.map.fitBounds(e.target.getBounds())
+    },
+    infoDiv (map) {
+      this._div = L.DomUtil.create('div', 'info') // create a div with a class "info"
```

Comment:
Would it also be possible to add this to the main HTML template and simply fetch it (like we do with the mapid div)?

Reply:
Not sure if necessary. The example code is already doing this job in a nice and clean way. But we can discuss it when refactoring the code.